jupyter_ydoc/ybasedoc.py

```diff
@@ -5,6 +5,7 @@
 from typing import Any, Callable, Dict, Optional
 
 from pycrdt import Doc, Map, Subscription, UndoManager
+from pycrdt_websocket.awareness import Awareness
```

Comment:
`jupyter-ydoc` cannot depend on `pycrdt-websocket`, I think we should bring the `Awareness` in `pycrdt`.

Reply:
Done in https://github.com/jupyter-server/jupyter_ydoc/pull/277/commits/3aa838774896786f35ab8314d933f2fcae684d3b